src/index.js

```diff
@@ -49,19 +49,22 @@ function parse(model, response, options = {}) {
 export default Bookshelf => {
   const Model = Bookshelf.Model.prototype;
   const client = Bookshelf.knex.client.config.client;
-  const parseOnFetch = client === 'sqlite' || client === 'sqlite3';
+  const handleOnSaving = client !== 'mysql';
```

Comment:
That's weird, if you see the test suite I added for MySQL, all tests passed on saving without registering a JSON column, which does not happen for other clients, and that's why we didn't tested against MySQL until now.

Can you provide a snippet with real data showing the plugin breaking? Turning on `debug` on the knexfile would help.